modules/core/src/main/scala/org/scalasteward/core/repoconfig/UpdatesConfig.scala

```diff
@@ -89,6 +93,109 @@ object UpdatesConfig {
   val defaultFileExtensions: Set[String] =
     Set(".scala", ".sbt", ".sbt.shared", ".sc", ".yml", "pom.xml")
 
+  private[repoconfig] val nonExistingFileExtension: List[String] = List(".non-exist")
```

Comment:
since current types are not distinguish between empty as all allowed and empty as nothing allowed - I've added artificial list

Reply:
Shouldn't we change the type of `fileExtensions` then? Does it even makes sense to have an empty list that is interpreted as "nothing allowed". That would mean Scala Steward can't edit any files. Who would want that?

Reply:
hypothetically if repo owner says pls change only `.scala` and `Scala Steward` sets to change only `.sbt` - you shouldn't have changes, right?

Reply:
About changing type - can we do it in another PR? I'm afraid it will bring a lot of changes to the code base since new structure will be
```scala
final case class UpdatesConfig(
    pin: Option[List[UpdatePattern]] = None,
    allow: Option[List[UpdatePattern]] = None,
    ignore: Option[List[UpdatePattern]] = None,
    limit: Option[PosInt] = None,
    includeScala: Option[Boolean] = None,
    fileExtensions: Option[List[String]] = None
)
```